src/styles/card.css

```diff
@@ -0,0 +1,86 @@
+.card {
+  font-family: "Roboto", Arial, sans-serif;
+  width: 200px;
+  padding-top: 32px;
+  padding-left: var(--horizontal-spacing);
+  padding-right: var(--horizontal-spacing);
+  padding-bottom: var(--vertical-spacing);
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__product-img {
+  width: 160px;
+  height: 134px;
```

Comment:
There is no need to set it one more time, as you set **width** and **height** on img tag